cms/middleware/toolbar.py

```diff
@@ -47,13 +47,34 @@ def toolbar_plugin_processor(instance, placeholder, rendered_content, original_c
     return output
 
 
+def get_client_ip(request):
```

Comment:
Perhaps, but please bear in mind that Django itself only checks the REMOTE_ADDR (https://github.com/django/django/blob/ecb59cc6579402b68ddfd4499bf30edacf5963be/django/contrib/admindocs/middleware.py#L20-L21).

Considering what this is used for, does it make sense to implement something as complex in CMS whereas Django itself uses only the simple check?